### PR TITLE
Refactor to multi-project

### PR DESCRIPTION
**Pull request by 2m**
Moves all common logic into AutoPlugin that is applied to every sub-project.

sub-project specific settings can be added to corresponding `build.sbt` file.

### CODE REVIEW

File: mqtt/build.sbt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+name := "akka-stream-contrib-mqtt"
+
+libraryDependencies ++= Seq(
+  "org.eclipse.paho" % "org.eclipse.paho.client.mqttv3" % "1.1.0"
```
Comment (patriknw) [inline mqtt/build.sbt]: please add a license note

**Comment by patriknw** [comment]
LGTM

**Comment by johanandren** [comment]
LGTM!

File: mqtt/build.sbt (hunk 1 above)
Comment (2m) [inline mqtt/build.sbt]: Will add license note with the next PR that adds initial mqtt connector.

**Pull request merged**